actions/app.go

```diff
@@ -3,47 +3,23 @@ package actions
 import (
 	"github.com/gobuffalo/buffalo"
 	"github.com/gobuffalo/buffalo/middleware"
-	"github.com/gobuffalo/buffalo/middleware/ssl"
 	"github.com/gobuffalo/envy"
-	"github.com/unrolled/secure"
-
-	"github.com/gobuffalo/x/sessions"
-	"github.com/rs/cors"
 )
 
-// ENV is used to help switch settings based on where the
-// application is being run. Default is "development".
 var ENV = envy.Get("GO_ENV", "development")
 var app *buffalo.App
 
-// App is where all routes and middleware for buffalo
-// should be defined. This is the nerve center of your
-// application.
 func App() *buffalo.App {
 	if app == nil {
 		app = buffalo.New(buffalo.Options{
-			Env:          ENV,
-			SessionStore: sessions.Null{},
-			PreWares: []buffalo.PreWare{
-				cors.Default().Handler,
-			},
-			SessionName: "_git2grid_session",
+			Env: ENV,
 		})
-		// Automatically redirect to SSL
-		app.Use(ssl.ForceSSL(secure.Options{
-			SSLRedirect:     ENV == "production",
-			SSLProxyHeaders: map[string]string{"X-Forwarded-Proto": "https"},
-		}))
-
-		// Set the request content type to JSON
-		app.Use(middleware.SetContentType("application/json"))
```

Comment:
I'd keep this middleware around.

Reply:
well the force ssl is not required because we can set the githook to https and also only open up 443. SetContentType("application/json") is also moot because githooks are sent with that header already.. was it added for testing maybe? Even then, it's a bit of a strange thing to on the server side. I figured i'd just trim these out to make things more predictable. What do you think?